modules/database/main.tf

```diff
@@ -34,6 +34,7 @@ resource "google_sql_database_instance" "default" {
     tier              = var.tier
     availability_type = var.availability_type
     user_labels       = var.labels
+    deletion_protection_enabled = true
```

Comment:
terraform fmt is going to be grumpy about this one

Reply:
Yikes, I did a `terraform fmt` and confidently pushed. Let me retry that

Reply:
Ok, this should look better now.